edelphi/src/main/java/fi/metatavu/edelphi/languagedetect/TextLanguageDetector.java

```diff
@@ -0,0 +1,64 @@
+package fi.metatavu.edelphi.languagedetect;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.api.client.repackaged.com.google.common.base.Optional;
+import com.optimaize.langdetect.LanguageDetector;
+import com.optimaize.langdetect.LanguageDetectorBuilder;
+import com.optimaize.langdetect.i18n.LdLocale;
+import com.optimaize.langdetect.ngram.NgramExtractors;
+import com.optimaize.langdetect.profiles.LanguageProfile;
+import com.optimaize.langdetect.profiles.LanguageProfileReader;
+import com.optimaize.langdetect.text.CommonTextObjectFactories;
+import com.optimaize.langdetect.text.TextObject;
+import com.optimaize.langdetect.text.TextObjectFactory;
+
+/**
+* This class is for detecting language from a text
+*
+* @author Ville Koivukangas
+*/
+public class TextLanguageDetector {
+	
+  private static TextLanguageDetector INSTANCE = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "INSTANCE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)